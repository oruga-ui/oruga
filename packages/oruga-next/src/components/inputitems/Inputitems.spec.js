import { shallowMount } from '@vue/test-utils'
import OInputitems from '@components/inputitems/Inputitems'

describe('OInputitems', () => {
    it('is called', () => {
        const wrapper = shallowMount(OInputitems)
        expect(wrapper.name()).toBe('OInputitems')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        const wrapper = shallowMount(OInputitems)
        expect(wrapper.html()).toMatchSnapshot()
    })


})
