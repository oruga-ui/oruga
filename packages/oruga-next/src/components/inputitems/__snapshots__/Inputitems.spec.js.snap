// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OInputitems render correctly 1`] = `
<div class="o-inputit">
    <div class="o-inputit__container">
        <o-autocomplete-stub usehtml5validation="true" statusicon="true" value="" data="" field="value" menuposition="auto" animation="fade" confirmkeys=",,Tab,Enter" rootclass="o-inputit__autocomplete" inputclasses="[object Object]"></o-autocomplete-stub>
    </div>
    <!---->
</div>
`;
